homework-submission/week2/src/components/TodoInput.js

```diff
@@ -0,0 +1,34 @@
+import React, { Component } from 'react';
+import './todoInput.css';
+
+export default class TodoInput extends Component {
+  constructor(props) {
+    super(props);
+    this.state = { value: this.props.todoText };
```

Comment:
you can write it like:
this.state = { value: "" };

so you don not need this.props.todoText